contrib/pzstd/Pzstd.cpp

```diff
@@ -217,14 +217,17 @@ static void compress(
  * @param numThreads The number of threads available to run compression jobs on
  * @param params     The zstd parameters to be used for compression
  */
-static size_t
-calculateStep(size_t size, size_t numThreads, const ZSTD_parameters& params) {
+static size_t calculateStep(
+    std::uintmax_t size,
+    size_t numThreads,
+    const ZSTD_parameters &params) {
   size_t step = 1ul << (params.cParams.windowLog + 2);
```

Comment:
This is just a minor question here, as I don't foresee any problem.

I read `1ul` as `(unsigned long) 1`, which may, or may not, be equivalent to `size_t`.
Any reason you prefer this to `(size_t)1` ?

The only issue I can think of is a potential conversion warning from some over-zealous compiler.